django_lightweight_queue/management/commands/queue_runner.py

```diff
@@ -85,6 +86,10 @@ def touch_filename(name):
         get_middleware()
         log.info("Loaded middleware")
 
+        # Configuration overrides
+        if num_only_queue_workers is not None:
+            app_settings.WORKERS[only_queue] = num_only_queue_workers
```

Comment:
We don't need to -- the validation of the command line flags means that if `num_only_queue_workers` is not `None` then we have a valid `only_queue`. I appreciate that that's a little indirect, but given that we remove `num_only_queue_workers` entirely in #15 I'm not worried.